projects/batfish/src/org/batfish/grammar/cisco/Cisco_ignored.g4

```diff
@@ -249,6 +249,7 @@ null_block_stanza
       | STCAPP
       | SVCLC
       | SWITCH
+      | SWITCHPORT
```

Comment:
If indeed 'switchport' appears at the top level (a unit test would be great for confirming this), then this keyword should be removed from null_block_substanza.